lib/rules/require-multiple-var-decl.js

```diff
@@ -99,11 +113,11 @@ module.exports = function() {};
 module.exports.prototype = {
     configure: function(options) {
         assert(
-            options === true || options === 'onevar',
-            this.getOptionName() + ' option requires a true value or `onevar`'
+            options === 'onevar' || options === 'nonconsecutive' || options === 'onedecl',
```

Comment:
Although this change in most probability will be landed to 2.0, it is better to preserve `true` value.